app/assets/stylesheets/tpi/_navbar.scss

```diff
@@ -0,0 +1,27 @@
+@import "bulma/sass/utilities/all";
+@import "app/sizes";
+@import "app/colors";
+
+.tpi--navbar {
+  .header-nav {
+    width: 100%;
+    background-color: $dark;
+  }
+
+  .navbar-item,
+  .navbar-link {
+    padding: 0.5rem 1.25rem;
+  }
+
+  .navbar-dropdown {
+    @if $desktop {
```

Comment:
haven't used @if conditionals in sass! is it like @media query?

Reply:
Yap. But `$desktop` is defined in bulma though. We only configure the breakpoints variables in `application.scss` (for now, at least).